components/list/src/provider/mod.rs

```diff
@@ -38,7 +38,6 @@ pub struct Baked;
 const _: () = {
     pub mod icu {
         pub use crate as list;
-        #[allow(unused_imports)] // baked data may or may not need this
```

Comment:
Please keep `#[allow(unused_imports)]` on icu_locid_transform (here and elsewhere)